shenyu-plugin/shenyu-plugin-oauth2/src/main/java/org/apache/shenyu/plugin/oauth2/OAuth2Plugin.java

```diff
@@ -70,8 +68,8 @@ public String named() {
 
     @Override
     public boolean skip(final ServerWebExchange exchange) {
-        Boolean skipStatus = exchange.<Boolean>getAttribute(OAUTH2_ENABLE);
-        return skipStatus == null || skipStatus;
+        Boolean skipStatus = exchange.<Boolean>getAttribute("enable");
+        return skipStatus == null || !skipStatus;
```

Comment:
`!Boolean.TRUE.equals(exchange.getAttribute("enable"))`

not enable is skip. 
That might make the meaning clearer